exonum/src/node/mod.rs

```diff
@@ -934,10 +978,26 @@ impl Node {
         };
         self.run_handler(&handshake_params)?;
 
-        // Stop all api handlers.
-        for mut handler in api_handlers {
-            handler.close().unwrap();
+        debug!("Handler stopped");
```

Comment:
1. IMHO handler stopping deserves `info` log level.
2. Handler stopping may be moved into separate method as well.